azure-pipelines.sds.qa-release.yml

```diff
@@ -0,0 +1,149 @@
+resources:
+  repositories:
+    - repository: azTemplates
+      type: github
+      name: hmcts/azure-devops-templates
+      ref: refs/heads/azure-dns-updates
+      endpoint: hmcts
+
+parameters:
+  - name: env
+    type: string
+    default: dev
+    values:
+      - dev
+
+  - name: qaenv
+    type: string
+    default: demo
+    values:
+      - demo
+
+  - name: git_repo_name
+    type: string
+    default: bookings-api
+
+trigger:
+  branches:
+    include:
+      - refs/tags/qa*
+
+variables:
+  - group: aks-cluster-info
+  - template: variables/shared.yaml
+    parameters:
+      env: ${{ parameters.env }}
+      git_repo_name: ${{ parameters.git_repo_name }}
+      cluster: $(cluster)
+  - name: prTagName
+    value: 'pr-379'
+  - name: dnsRecordName
+    value: ${{ parameters.git_repo_name }}-${{ variables.prTagName }}
+
+stages:
+  - stage: Uninstall
+    displayName: Uninstall from aks-$(cluster)-${{ parameters.env }}
+    jobs:
+      - job:
+        steps:
+          - template: templates/Containerisation/helm/helm-remove.yaml@azTemplates
+            parameters: 
+              helmVersion: $(chartVersion)
+              azureSubscription: ${{ variables.subscriptionName }}
+              kubernetesRG: "${{ variables.clusterResourceGroup }}"
+              kubernetesCluster: "${{ variables.clusterName }}"
+              chartPath: '${{ variables.chartPath }}'
+              namespace: "vh"
+              chartName: '${{ variables.chartName }}'
+              imageTag: '${{ variables.prTagName }}'
+              environment: ${{ parameters.env }}
+          - template: templates/Azure/Dns/private-dns-remove.yaml@azTemplates
+            parameters: 
+              subscriptionName: ${{ variables.subscriptionName }}
+              dnsSubscriptionName: ${{ variables.dnsZoneSubscription }}
+              dnsResourceGroupName: ${{ variables.dnsZoneResourceGroup }}
+              zoneName: ${{ variables.dnsZone }}
+              recordName: ${{ variables.dnsRecordName }}
+
+  - stage: Install
```

Comment:
Should we template this as it is the same for Dev release and QA release?

Reply:
yep, probably!

Reply:
done